app/models/comment.rb

```diff
@@ -1,5 +1,5 @@
 class Comment < ApplicationRecord
-  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
   belongs_to :post, class_name: 'Post', foreign_key: 'post_id', counter_cache: true
 
   after_save :update_commentcounter
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍